### ISSUE TITLE
Error connecting the display with a macro lazy static

### ISSUE THREAD

**Issue by vohebup** [vberger/wayland-client-rs#20]
```Rust
#[macro_use]
extern crate lazy_static;
extern crate wayland_client as wayland;

use wayland::wayland::WlDisplay;

lazy_static!{
    static ref DISPLAY: WlDisplay = {
        match wayland::wayland::get_display() {
            Some(d) => d,
            None => panic!("Unable to connect to a wayland compositor.")
        }
    };
}

fn main() {
    println!("Hello, World!");

}
```
Compiler error:
```
<lazy_static macros>:33:22: 33:34 error: the trait `core::marker::Sync` is not implemented for the type `*mut wayland_sys::client::wl_proxy` [E0277]
<lazy_static macros>:33 = __stability (  ) ; require_sync ( static_ref ) ; static_ref } } }
                                             ^~~~~~~~~~~~
<lazy_static macros>:4:1: 5:73 note: in this expansion of lazy_static! (defined in <lazy_static macros>)
src/main.rs:7:1: 14:2 note: in this expansion of lazy_static! (defined in <lazy_static macros>)
<lazy_static macros>:33:22: 33:34 help: run `rustc --explain E0277` to see a detailed explanation
<lazy_static macros>:33:22: 33:34 note: `*mut wayland_sys::client::wl_proxy` cannot be shared between threads safely
<lazy_static macros>:33:22: 33:34 note: required because it appears within the type `wayland::sys::wayland::client::WlDisplay`
<lazy_static macros>:33:22: 33:34 note: required by `DISPLAY.::std::ops::Deref::deref::require_sync`
error: aborting due to previous error
Could not compile `wayland_ffi`.

To learn more, run the command again with --verbose.
```

**Comment by vberger** [comment]
This is indeed an oversight from my side.

The wayland library has pretty good guarantees regarding thread-safety. I'll need to double-check that everything is ok and add the `Send` and `Sync` impls to the wayland types.

**Comment by vberger** [comment]
I've published wayland-scanner `0.1` that should fix it. You should just need to run a `cargo update` for now.

I'll update the dependencies of wayland-client for the next version.
